app/common/src/map/PriceMarker.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+
+import Color from '../Color';
+
+const styles = StyleSheet.create({
+  container: {
+    // price marker is wider than default, so needs to be shifted more to the left
+    marginLeft: -30,
+    marginTop: -10,
```

Comment:
I would not do that, it's very problematic (at least not that easy) and we are not able to do it properly now. It doesn't work if you are rotating the map, it doesn't work if you are changing angle (2D -> 3D map) so I cannot really see the improvement. Can we just keep it "not working" without margin? 🙂 Let's fix it properly later if needed.